core/build.gradle

```diff
@@ -1,9 +1,9 @@
 plugins {
     id("com.android.library")
-    kotlin("android")
+    id("kotlin-android")
 }
 
-val composeVersion = "1.2.0-alpha07"
+def composeVersion = "1.2.0-alpha07"
```

Comment:
Is there any reason we've migrated from kotlin gradle builds to groovy?

Reply:
Only because of my lack of gradle knowledge, basically I couldn't do the kts version of `apply from` https://stackoverflow.com/a/55337202